tools/check_tidy.py

```diff
@@ -28,6 +30,16 @@
 from common_py.system.executor import Executor as ex
 
 
+def parse_option():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--autoedit', action='store_true', default=False,
+        help='edits to files instead of displaying a diff' +
```

Comment:
Maybe a bit better help message would be: `Automatically edit/fix the detected clang format problems. No diff(s) will be created/displayed.`